MusiciansAPP/MusiciansAPP.BL.UnitTests/ArtistsService/Logic/ArtistsServiceTests.cs

```diff
@@ -0,0 +1,131 @@
+using AutoMapper;
+using Moq;
+using MusiciansAPP.BL.ArtistsService.BLModels;
+using MusiciansAPP.DAL.DALModels;
+using MusiciansAPP.DAL.DBDataProvider.Interfaces;
+using MusiciansAPP.DAL.WebDataProvider.Interfaces;
+using MusiciansAPP.Domain;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MusiciansAPP.BL.UnitTests.ArtistsService.Logic;
+
+[TestFixture]
+public class ArtistsServiceTests
+{
+    private const int TotalItems = 3;
+    private const int PageSize = 2;
+    private const int Page = 1;
+    private const string ArtistOneName = "a";
+    private const string ArtistOneImageUrl = "aa";
+    private const string ArtistTwoName = "b";
+    private const string ArtistTwoImageUrl = "bb";
+
+    private BL.ArtistsService.Logic.ArtistsService _service;
+    private Mock<IWebDataProvider> _webDataProvider;
```

Comment:
Incorrect naming. It isn't _webDataProvider, it's _webDataProviderMock.
Please fix it